### DISCUSSION TITLE
Poll: Unconference Styles for 2024 Festival

### DISCUSSION THREAD

**Discussion by julianpistorius** [resbazaz/festival2024#20] [Polls]
**Goal of the Poll**

ResBaz Arizona 2024, from May 15th to 17th, is an interactive conference focusing on digital research skills. It's a place to learn, share, network, and have fun, catering to researchers and people who work with data at all levels.

This poll helps the Research Bazaar Arizona group choose the best ways to run the unconference day of our upcoming event. Your choices will make the event fit what we all want and encourage working together in a lively way.

**What is an Unconference?**

An unconference is a meeting led by its participants, different from usual conferences. People attending decide what to talk about, work together in workshops, and focus more on discussions than presentations. It's about using everyone's ideas and experiences.

See [unConferencing – how to prepare to attend an unconference](https://unconference.net/unconferencing-how-to-prepare-to-attend-an-unconference-2/).

**Lessons from ["Ten Simple Rules for Organizing an Unconference"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4310607/)**

1. **Decide the Event Format:** Choose between a traditional conference or an unconference based on goals, participant number, and desired interaction level.
2. **Right Format for Goals:** Select a format that aligns with your mission, whether it be open discussions, hackathons, or curated sessions.
3. **Clear Mission:** A defined mission focuses the content and structure, aligning with participants' goals.
4. **Minimize Lectures:** Encourage cooperative knowledge exchange over one-directional presentations.
5. **Participant Involvement:** Engage attendees in planning to foster investment and enthusiasm.
6. **Open, Relaxed Atmosphere:** Facilitate a welcoming and respectful environment conducive to interaction.
7. **Trust Your Community:** Share leadership with participants to encourage a sense of ownership and empowerment.
8. **Effective Communication:** Utilize various tools for clear, transparent, and collaborative communication.
9. **Value the Journey:** Focus on collaborative process and documentation, not just final outcomes.
10. **All Ideas Matter:** Encourage and value contributions at all levels, fostering a rich brainstorming environment.

**Unconference Facilitation Styles**

:information_source: **_Note: These are not mutually exclusive!_** As @val-pf [mentions in a comment below](https://github.com/resbazaz/festival2024/discussions/20#discussioncomment-9002836), if there are multiple popular options we could have two to five of these in a full day.

1. A **"birds-of-a-feather session"** refers to an informal discussion group at a conference, where people with a shared interest gather without a pre-set agenda. This format allows attendees to spontaneously form groups around specific research interests or digital tools, fostering organic conversations and networking opportunities. For more details, you can visit the [Birds-of-a-feather session definition on Wiktionary](https://en.wiktionary.org/wiki/birds-of-a-feather_session). 

2. **Dot-voting, also known as dotmocracy**, is a facilitation method where participants use a limited number of dot stickers or marks to vote on presented options. This process allows multiple options to be quickly prioritized and is less cognitively demanding than ranking all options. It can foster engagement and transparency in decision-making. In ResBazAZ, dot-voting could be used to determine the most popular topics or workshops, allowing attendees to have a direct impact on the agenda or activities. For more details, you can visit the [Dot-voting page on Wikipedia](https://en.wikipedia.org/wiki/Dotmocracy). 

3. A **fishbowl conversation** is a dialog format suitable for large groups, often used in participatory events such as unconferences. It involves a small group of participants in an inner circle (the 'fishbowl'), discussing a topic, while the rest of the participants sit outside and listen. In an open fishbowl, an empty chair is left for any audience member to join the conversation, while in a closed fishbowl, a set group speaks before leaving for a new group. This method promotes democratic participation and equalizes the roles of speakers and listeners, making it a great fit for ResBazAZ's inclusive and interactive atmosphere. For more details, you can visit the [Fishbowl conversation page on Wikipedia](https://en.wikipedia.org/wiki/Fishbowl_(conversation)).

4. **Ignite events** feature speakers who have five minutes to talk on a subject, accompanied by 20 slides that automatically advance every 15 seconds. This format encourages concise, engaging presentations. In the ResBazAZ context, Ignite could be a dynamic way for researchers to present their work or ideas quickly and effectively, fostering a lively exchange of ideas and stimulating rapid knowledge sharing. For more details, you can visit the [Ignite event page on Wikipedia](https://en.wikipedia.org/wiki/Ignite_(event)).

5. **Lightning talks** are brief presentations, usually around five minutes, offered in quick succession during a conference segment. This format requires speakers to be concise, allowing them to communicate their main points efficiently and enabling a wide range of topics to be covered in a short time. For ResBazAZ, lightning talks could be a great way for participants to share their research findings or insights on digital tools quickly, making it possible to expose attendees to a broad array of ideas and projects in a limited timeframe. For more details, you can visit the [Lightning talk page on Wikipedia](https://en.wikipedia.org/wiki/Lightning_talk).

6. **PechaKucha** is a storytelling format where presenters show 20 slides for 20 seconds each, totaling a presentation time of 6 minutes and 40 seconds. It's a way for individuals to share personal presentations about their work in a concise and visually engaging manner. At ResBazAZ, PechaKucha could provide researchers a platform to showcase their work or interests in a structured yet creative way, encouraging clear and impactful communication. For more details, you can visit the [PechaKucha page on Wikipedia](https://en.wikipedia.org/wiki/PechaKucha).

7. **Speed geeking** is a method where participants can quickly experience several presentations within a set time. Presenters are positioned around a room, and small groups of audience members rotate among them. Each presenter has about five minutes for their presentation and Q&A before the audience moves to the next presenter. This format is similar to speed dating and is useful in events like ResBazAZ to allow for rapid, diverse exposure to a wide range of topics or research in a short amount of time. For more details, you can visit the [Speed geeking page on Wikipedia](https://en.wikipedia.org/wiki/Speed_geeking).

8. **The World Café** is a structured conversational process designed for knowledge sharing, where participants discuss a topic at several small tables, akin to a café. The process facilitates open discussion without pre-defined outcomes, and is effective for group sizes of twelve or more. For ResBazAZ, World Café could be a powerful tool to facilitate in-depth discussions on research topics, allowing participants to contribute and cross-pollinate ideas in an informal, café-like setting. For more details, you can visit the [World café (conversation) page on Wikipedia](https://en.wikipedia.org/wiki/World_caf%C3%A9_(conversation)).

A 'meta style' is **Open Space Technology (OST)** - a self-organizing meeting method used in conferences and multi-day events. Unlike traditional conferences with pre-set schedules, OST sessions are determined by the participants at the event, creating a more dynamic and engaging atmosphere. This approach could be helpful if there are multiple popular options below. Then we can use OST to schedule the day to include the options to fit. For more details, you can visit the [Open Space Technology page on Wikipedia](https://en.wikipedia.org/wiki/Open_Space_Technology).

**Comment by jcoliver** [comment]
I want to vote for them all :rofl:

**Reply by julianpistorius** [reply]
Yes... I wish GitHub polls had multiple choice options. :thinking:

**Comment by val-pf** [comment]
these are not mutually exclusive, right? We could have two to five of these in a full day?

**Reply by julianpistorius** [reply]
Correct. Also see the OST blurb (I highlighted the relevant part):

> A 'meta style' is Open Space Technology (OST) - a self-organizing meeting method used in conferences and multi-day events. Unlike traditional conferences with pre-set schedules, OST sessions are determined by the participants at the event, creating a more dynamic and engaging atmosphere. _**This approach could be helpful if there are multiple popular options below. Then we can use OST to schedule the day to include the options to fit.**_ For more details, you can visit the [Open Space Technology page on Wikipedia](https://en.wikipedia.org/wiki/Open_Space_Technology).

**Reply by julianpistorius** [reply]
I added a bit of text to indicate that they are not mutually exclusive. Thank you @val-pf!

**Comment by lwdozal** [comment]
These are so fun!!  I'm wondering about the logistics of the presentations within a session (pecha kucha, speed geeking, lightning talk). Would we have to ask for speakers ahead of time? If so, we need to get on promoting this!

**Reply by lwdozal** [reply]
Or do we ask people to just show up with a presentation if they want to participate?

**Comment by brandon-jernigan** [comment]
@julianpistorius we were talking at hacky hour, it might be good to wrap up this discussion by the beginning of next week so we can move forward with emailing volunteers. What do you think?

**Comment by rbartelme** [comment]
I agree with @brandon-jernigan. I think, based upon the poll, we'll probably end up with birds of a feather and, thematically, it seems people want some sort of rapid fire talk format. Maybe that should be the focus? But we'll let the poll run its course.

**Reply by hidyverse** [reply]
I agree with @brandon-jernigan and @rbartelme. Looks like we can group these events into 1) socials and 2) rapid-fire talks so let's go forward with that. 

I think Birds of a feather is more recognizable than world cafe and lightening talks is more recognizable than PechaKucha and speed geeking. Not sure recognizable is the only selling point, but folks may be more encouraged to attend events they have heard of

**Reply by brandon-jernigan** [reply]
I was also thinking Speed Geeking could be reformatted to 1-1 conversations between people so people can get to know each other

**Comment by hidyverse** [comment]
What if we add a question to the registration form about which session styles participants are interested to attend?

**Comment by val-pf** [comment]
I think we discussed adding a question on topics for unconference, too. Maybe both?

**Reply by hidyverse** [reply]
Yes, let's include at least BoF, lightning talks, and speed geeking? 
Then we can prompt for a topic for any of those options.
